lib/src/main/java/com/dynatrace/metric/util/Metric.java

```diff
@@ -262,7 +265,9 @@ public Builder setCurrentTime() {
      * @return A {@link String} containing all properties set on the {@link Metric.Builder} in an
      *     ingestion-ready format.
      * @throws MetricException If no value is set or if the prefix/metric key combination evaluates
-     *     to an invalid/empty metric key after normalization.
+     *     to an invalid/empty metric key after normalization. Will also throw a {@link
+     *     MetricException} when the line length after serialization exceeds the maximum line length
+     *     (2000 characters).
```

Comment:
```suggestion
     *     accepted by the ingest API.
```
The fact that it's 2000 is an implementation detail and specifying it here would make that part of the API and thus changing it a breaking change since users could depend on it.